src/ansiblelint/constants.py

```diff
@@ -1,9 +1,13 @@
 """Constants used by AnsibleLint."""
 
+import re
 from enum import Enum
 from pathlib import Path
 from typing import Literal
 
+TAG_NAME_REGEXP = re.compile(r"^(?!.*__)[a-z]+[0-9a-z_]*$")
```

Comment:
```suggestion
TAG_NAME_REGEXP = re.compile(r"^(?!.*__)[a-z][0-9a-z_]*$")
```

I think removing the `+` here should avoid the regex issues being reported, while matching the same strings.

Reply:
What regex errors? This was an exact copy and paste from galaxy importer lib.